### PR TITLE
Action Buttons that manipulate a form are only visible to the owner

### LINKED ISSUE #80: hide action buttons from non-owner of form
A non-owner cannot manage a form of other user, yet action buttons are nonetheless visible (the backend is handling the lack of permissions correctly). The buttons should not be visible in the UI.

### PR DESCRIPTION
**Pull request by PascalinDe**
This PR makes sure that action buttons to manipulate a form are only visible to the owner of the form.

closes #80

### CODE REVIEW

File: web/frontend/tests/forms.spec.ts [hunk 1]
```diff
@@ -74,3 +81,77 @@ async function assertIsOnlyVisibleToOwnerStates(page: page, locator: locator, st
 test('Assert "Add voters" button is only visible to owner', async ({ page }) => {
   await assertIsOnlyVisibleToOwnerStates(page, page.getByTestId('addVotersButton'), [0, 1]);
 });
+
+test('Assert "Initialize" button is only visible to owner', async ({ page }) => {
```
Comment (ineiti) [inline web/frontend/tests/forms.spec.ts:85]: Lazy review: are there positive tests on the buttons where it checks that the buttons are available?

**Review by ineiti — APPROVED** [review]
Just one questions...

File: web/frontend/tests/forms.spec.ts (hunk 1 above)
Comment (PascalinDe) [reply web/frontend/tests/forms.spec.ts:85]: not yet, that's why it closes #80 and not #72 :slightly_smiling_face:

Comment (PascalinDe) [reply web/frontend/tests/forms.spec.ts:85]: or at least not all of them - e.g. it checks that the buttons that are should only be visible to owner are:

* not visible to unauthenticated
* not visible to non-owner user
* not visible to non-owner admin
* visible to owner

but it does not e.g. check that the "Vote" button is visible for voters

Comment (PascalinDe) [reply web/frontend/tests/forms.spec.ts:85]: also it checks only in specific states, e.g. it does not verify that the buttons are not displayed in other states where they are not applicable at all

**Pull request merged**